warehouse/models/gtfs_views/gtfs_schedule_dim_shapes_geo_latest.sql

```diff
@@ -6,7 +6,8 @@ WITH gtfs_schedule_dim_shapes_geo AS (
 ),
 
 gtfs_schedule_dim_shapes_geo_latest AS (
-    SELECT *
+    -- TODO: we should probably timestamp these in some way
```

Comment:
We probably want to tell downstream consumers when the data was generated/captured in some way. I don't necessarily know if we want to use `calitp_extracted_at` since it's funky.